### ISSUE TITLE
[BUG] Apparently erroneous runtime error when sandboxed

### ISSUE THREAD

**Issue by mattgodbolt** [compiler-explorer/compiler-explorer#1675]
Executing this code: https://godbolt.org/z/_WhHNF gives an error:

```
ASM generation compiler returned: 0
Execution build compiler returned: 0
Program returned: 0
example.cpp:10:18: runtime error: member call on address 0x00000064aa40 which does not point to an object of type 'std::basic_ostream<char>'
0x00000064aa40: note: object has invalid vptr
<memory cannot be printed>
10
```

**To Reproduce**
Follow the link

**Expected behavior**
Just 10 to be printed. Running on admin node:

```
Nov/08 20:03 admin-node/tmp 2.1s $ /opt/compiler-explorer/clang-9.0.0/bin/clang++ -g -o output.s -mllvm --x86-asm-syntax=intel  --gcc-toolchain=/opt/compiler-explorer/gcc-9.2.0 -fcolor-diagnostics -fno-crash-diagnostics -isystem/celibs/boost_1_71_0 -std=c++14 ./test.cc 
Nov/08 20:03 admin-node/tmp 1.3s $ ./output.s 
10
```

I haven't tried running in the sandbox, but I suspect it's related to that.

**Comment by partouf** [comment]
similar to https://github.com/mattgodbolt/compiler-explorer/issues/1636

**Comment by apmorton** [comment]
nsjail fixed this issue
